spec/checkPullRequestJobSpec.js

```diff
@@ -478,25 +721,29 @@ describe('Pull Request Job Spec', () => {
     it('should ping server jobs admin', () => {
       expect(github.issues.createComment).toHaveBeenCalled();
       const author = payloadData.payload.pull_request.user.login;
-      const formText = 'server jobs form'.link(
-        'https://goo.gl/forms/XIj00RJ2h5L55XzU2');
+      const formText = (
+        'server jobs form'.link('https://goo.gl/forms/XIj00RJ2h5L55XzU2'));
```

Comment:
Just curious, was this a recommended change from eslint?

Reply:
https://github.com/oppia/oppiabot/pull/148#discussion_r511668220

Reply:
Okay.